public/javascripts/Gallery/src/displays/SeverityDisplay.js

```diff
@@ -26,36 +32,62 @@ function SeverityDisplay(container, severity, isModal=false) {
         if (isModal) {
             // Add bold weight. Find better way to do this.
             title.classList.add('modal-severity-header');
+            // centers tooltip
+            holder.classList.add('modal-no-severity-box')
         }
 
         title.innerText = `${i18next.t("severity")}`;
+        // if no severity rating, gray out title
+        if (unsupported || !severity) {
+            title.classList.add('no-severity-header')
+        }
         container.append(title);
 
-        // Creates all of the circles for the severities.
+        // Highlight the correct severity.
+        // We do so by darkening a number of circles from the left equal to the severity. For example, if the severity
+        // is 3, we will darken the left 3 circles.
         for (let i = 1; i <= 5; i++) {
             let severityCircle = isModal ? new Image() : document.createElement('div');
             severityCircle.className = severityCircleClass;
-            if (isModal) {
-                // Set the src of our smiley icon to default black-outlined, white-filled smileys.
-                severityCircle.src = `/assets/javascripts/SVLabel/img/misc/SmileyRating_${i}_gallery.png`;
-            }
 
-            circles.push(severityCircle);
-        }
-
-        // Highlight the correct severity.
-        // We do so by darkening a number of circles from the left equal to the severity. For example, if the severity
-        // is 3, we will darken the left 3 circles.
-        // TODO: rename these once confirmed. also, we can probably move this to the upper loop.
-        if (severity) {
-            for (let i = 0; i < severity; i++) {
+            if (unsupported || !severity) {
```

Comment:
Can we switch to checking specifically if `severity` is null or undefined (not sure which it is in this case). I think I'd prefer that over checking if it is falsey